src/checkout/checkout-service.ts

```diff
@@ -1052,6 +1054,25 @@ export default class CheckoutService {
         return this._dispatch(action);
     }
 
+    /**
+     * Initializes the spam protection for order creation.
+     *
+     * The order creation will fail if spam protection is enabled but
```

Comment:
this is the documentation the developer gets, maybe explain a bit more what the spam protection is about?

Reply:
👌 Will add more details on this.